tests/foreman/ui/test_contenthost.py

```diff
@@ -172,7 +172,7 @@ def test_positive_end_to_end(session, repos_collection, vm):
     """Create all entities required for content host, set up host, register it
     as a content host, read content host details, install package and errata.
 
-    :id: f43f2826-47c1-4069-9c9d-2410fd1b622c
+    :id: d2bd7166-7dc2-41ae-bf6b-590f933501d9
```

Comment:
Is it a duplicate ID ? Why its changed ?

Reply:
Removed this accidental change.